airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/async/model/PartialAirbyteMessage.kt

```diff
@@ -26,6 +27,12 @@ class PartialAirbyteMessage {
     @JsonProperty("state")
     var state: PartialAirbyteStateMessage? = null
 
+    @get:JsonProperty("trace")
```

Comment:
I have no idea if these are even necessary, literally just copypasting from other fields. Feels like they shouldn't be? since `trace` doesn't have any spaces/underscores/etc

Reply:
Not needed but leave it for consistency because I never understood that syntax (auto-conversion did it)